app/models/desfake/itinerary.rb

```diff
@@ -8,19 +8,27 @@ class Itinerary < ApplicationRecord
     include Decidim::HasUploadValidations
     include Decidim::TranslatableResource
 
-    translatable_fields :title, :subtitle, :description
+    translatable_fields :title, :subtitle, :description, :body
```

Comment:
What's the difference between `description` and `body`? I see `description` on the itinerary's card and body on the itinerary's show page but I think they are the same attribute.

If the client wants a different text on the card and the show page I said nothing.

Reply:
I think that's exactly what the client wants ;)

The `show` page will have formatted text (`body`), whereas the card's `description` should be plain text.